### PR TITLE
Centralized CMS caching 2

### PR DESCRIPTION
**Pull request by anthonyshull**
https://app.asana.com/0/555089885850811/1206129599937934/f

### CODE REVIEW

File: apps/cms/lib/repo.ex [hunk 1]
```diff
@@ -50,20 +56,19 @@ defmodule CMS.Repo do
 
   @spec news_entry_by(Keyword.t()) :: NewsEntry.t() | :not_found
   def news_entry_by(opts) do
```
Comment (kotva006) [inline apps/cms/lib/repo.ex:58]: Is there a reason this page isn't cached?

Comment (anthonyshull) [reply apps/cms/lib/repo.ex:58]: Yeah, this goes to a "view" in Drupal. We don't have a mechanism for the CMS to tell us that a view has been updated. Though, I think it's possible. So, for now we're removing the caching.

**Comment by anthonyshull** [comment]
> The changes here look fine. Though it would be nice if this was pointing at the first pull request seeing as the changes are duplicated here (and aren't reflecting the feedback from the first pull request)

The differences will shake out when the first PR is merged into master.

**Comment by thecristen** [comment]
Complete in #1856

**Pull request closed without merging**